mobly/test_runner.py

```diff
@@ -247,7 +247,7 @@ def is_testfile_name(name, ext):
                         # under py2, because "raise X from Y" syntax is only
                         # supported under py3.
                         self.log.exception(msg)
-                        raise USERError(msg)
+                        raise ValueError(msg)
```

Comment:
In general YAGNI unless you think it's an important use case, but I'm ok with leaving proper removal for a later PR :)